tests/framework/utils/env.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"os"
+)
+
+func IsEnvVarPresent(env string) bool {
```

Comment:
How about adding all these env helpers in tests/framework/installer/environment.go? Or does that cause a circular dependency?

Reply:
We don't need this anymore so removed this method but yes, earlier it was creating circular dependency.